homeassistant/components/qbittorrent/sensor.py

```diff
@@ -117,24 +145,113 @@ def update(self):
         except self._exception:
             _LOGGER.error("Invalid authentication")
             return
-
         if data is None:
             return
-
+        attributes = {}
         download = data["server_state"]["dl_info_speed"]
         upload = data["server_state"]["up_info_speed"]
 
         if self.type == SENSOR_TYPE_CURRENT_STATUS:
             if upload > 0 and download > 0:
                 self._state = "up_down"
+                self.attrs = attributes
```

Comment:
Can't you just move this line out side of these `if` statements and just do it at the end once?